src/components/App/App.js

```diff
@@ -18,19 +18,13 @@ function App() {
     <Router>
       <ErrorBoundary>
         <Switch>
-          <Route path="/" exact>
-            <IsLoggedIn>
-              <Models />
-            </IsLoggedIn>
-          </Route>
-          <Route path="/models/*" exact>
-            <IsLoggedIn>
-              <ModelDetails />
-            </IsLoggedIn>
-          </Route>
-          <Route path="/controllers" exact component={Controllers} />
-          <Route path="/usage" exact component={Usage} />
-          <Route path="/logs" exact component={Logs} />
+          <Login>
```

Comment:
This will block the routes from being defined until after the login has happened...Can this introduce any race conditions or other issues?

Reply:
@barrymcgee and I discussed offline that while this works as intended now, we'll keep an eye out for any potential race conditions in the future and investigate if this architecture can introduce any unintended side effects.